### PR TITLE
Fix cookie domain and direct sign in link to DGA

### PR DESCRIPTION
**Pull request by AlexGilleran**
### What this PR does
- Stops the noPreview cookie being on the wrong domain
- Deletes the VWO cookies on redirect

### Checklist
- [x] Unit tests aren't applicable (delete one)
- [x] I've updated CHANGES.md with what I changed.
- [x] I've linked this PR to an issue in ZenHub (or there is no issue) and dragged it to the _Pull Request_ column

### CODE REVIEW

File: magda-web-client/src/UI/Banner.js [hunk 1]
```diff
@@ -14,7 +14,24 @@ class Banner extends React.Component {
...
         event.preventDefault();
-        document.cookie = "noPreview=true; path=/";
+
+        if (window.location.hostname === "search.data.gov.au") {
+            // Delete all VWO's cookies
+            const cookieNames = document.cookie.split(/=[^;]*(?:;\s*|$)/);
+            // Remove any that match the pattern
+            for (var i = 0; i < cookieNames.length; i++) {
+                if (/^_(vis|vwo).*/.test(cookieNames[i])) {
+                    console.log(cookieNames[i]);
```
[the remark this answers is not in the document]
Comment (AlexGilleran) [reply magda-web-client/src/UI/Banner.js]: Ah jeez, yeah should be no console.log :(

File: magda-web-client/src/UI/Banner.js [hunk 2]
```diff
@@ -14,7 +14,24 @@ class Banner extends React.Component {
 
     goBack = event => {
         event.preventDefault();
-        document.cookie = "noPreview=true; path=/";
+
+        if (window.location.hostname === "search.data.gov.au") {
```
Comment (t83714) [inline magda-web-client/src/UI/Banner.js]: Do we want to use a cookie lib? e.g. js-cookie?
I can find any issue with your code by looking at it.
The only issue is after a couple tests. my cookie state became a little bit wired:
image
And it caused next test failed as it split `"NaN; _vwo_uuid_v2"` into a name

File: magda-web-client/src/UI/Banner.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (maxious) [reply magda-web-client/src/UI/Banner.js]: I did polyfill console.log but a good habit to take them out if you don't need them ;)

File: magda-web-client/src/UI/Banner.js (hunk 2 above)
Comment (AlexGilleran) [reply magda-web-client/src/UI/Banner.js]: Switched to js-cookie 🍪

Comment (t83714) [reply magda-web-client/src/UI/Banner.js]: Thank you!
Tested by coding injection (running in chrome js console )
Works fine

**Pull request merged**